sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientTest.java

```diff
@@ -290,6 +307,37 @@ void sendSingleMessage() {
         Assertions.assertEquals(Section.SectionType.Data, message.getBody().getType());
     }
 
+    @Test
+    void scheduleMessage() {
+        // Arrange
+        long sequenceNumberReturned = 10;
+        Instant instant = mock(Instant.class);
+
+        when(connection.createSendLink(eq(ENTITY_NAME), eq(ENTITY_NAME), any(AmqpRetryOptions.class)))
+            .thenReturn(Mono.just(sendLink));
+        when(sendLink.getLinkSize()).thenReturn(Mono.just(MAX_MESSAGE_LENGTH_BYTES));
+        when(managementNode.schedule(eq(message), eq(instant), any(Integer.class)))
+            .thenReturn(just(sequenceNumberReturned));
+
+        // Act & Assert
+        StepVerifier.create(sender.scheduleMessage(message, instant))
+            .expectNext(sequenceNumberReturned)
+            .verifyComplete();
+    }
```

Comment:
I could have been more clear, the verification of management node operation should be here as well.